TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithSelectStar.rdzSQL.txt

```diff
@@ -0,0 +1,28 @@
+﻿--- Sql Statements ---
+line6:SelectStatement
```

Comment:
Just add a space after 'line' and another space after ':', this is more readable